jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/resources/org/eclipse/lsp4jakarta/jdt/core/messages/messages.properties

```diff
@@ -121,6 +121,7 @@ RemoveAllEntityParametersExcept = Remove all entity parameters except {0}
 ErrorMessageJsonbCreator = Only one constructor or static factory method can be annotated with @JsonbCreator in a given class.
 ErrorMessageJsonbTransientOnField = When a class field is annotated with @JsonbTransient, this field, getter or setter must not be annotated with other JSON Binding annotations.
 ErrorMessageJsonbTransientOnAccessor = When an accessor is annotated with @JsonbTransient, its field or the accessor must not be annotated with other JSON Binding annotations.
+ErrorMessageJsonbPropertyUniquenessField = Multiple fields or properties with @JsonbProperty must not have JSON members with duplicate names, which violates the property uniqueness.
```

Comment:
"which violates the property uniqueness" should instead be "the member names must be unique"